File: FingertipsProfileManager/ProfileData/Entities/Report/Report.cs
using Newtonsoft.Json;

namespace Fpm.ProfileData.Entities.Report
{
    public class Report
    {
        [JsonProperty(PropertyName = "id")]
        public int Id { get; set; }
        [JsonProperty(PropertyName = "name")]
        public string Name { get; set; }
        [JsonProperty(PropertyName = "file")]
        public string File { get; set; }
        [JsonProperty(PropertyName = "parameters")]
        public string Parameters { get; set; }
        [JsonProperty(PropertyName = "notes")]
        public string Notes { get; set; }
        [JsonProperty(PropertyName = "isLive")]
        public bool IsLive { get; set; }
        [JsonProperty(PropertyName = "areaTypeIds")]
        public string AreaTypeIds { get; set; }
    }
}

File: FingertipsProfileManager/MainUI/angular-app/src/app/model/report.ts
import { Profile, AreaType } from './profile';

export class Report {
    public id: number = null;
    public name = '';
    public file = '';
    public profiles: number[] = [];
    public parameters: string[] = [];
    public notes: string;
    public isLive: boolean;
    public areaTypeIds: number[] = [];
}

export class ReportListView {
    public id: number = null;
    public name = '';
    public file = '';
    public profiles: Profile[] = [];
    public parameters: string[] = [];
    public notes: string;
    public isLive: boolean;
    public areaTypes: AreaType[] = [];
}
